dotnet/src/Functions/Functions.Prompty/Extensions/PromptyKernelExtensions.cs

```diff
@@ -20,21 +20,39 @@ public static class PromptyKernelExtensions
     /// Create a <see cref="KernelFunction"/> from a prompty file.
     /// </summary>
     /// <param name="kernel">kernel</param>
-    /// <param name="promptyPath">path to prompty file.</param>
+    /// <param name="promptyFilePath">Prompty template.</param>
     /// <param name="promptTemplateFactory">prompty template factory, if not provided, a <see cref="LiquidPromptTemplateFactory"/> will be used.</param>
     /// <param name="loggerFactory">logger factory</param>
     /// <returns><see cref="KernelFunction"/></returns>
     /// <exception cref="ArgumentNullException"></exception>
     /// <exception cref="NotSupportedException"></exception>
-    public static KernelFunction CreateFunctionFromPrompty(
+    public static KernelFunction CreateFunctionFromPromptyFile(
         this Kernel kernel,
-        string promptyPath,
+        string promptyFilePath,
         IPromptTemplateFactory? promptTemplateFactory = null,
         ILoggerFactory? loggerFactory = null)
     {
-        Verify.NotNull(kernel);
+        var promptyTemplate = File.ReadAllText(promptyFilePath);
```

Comment:
```suggestion
        Verify.NotNull(kernel);
        Verify.NotNull(promptyFilePath);

        var promptyTemplate = File.ReadAllText(promptyFilePath);
```

Reply:
It should probably be `Verify.NotNullOrWhitespace(promptyFilePath);` here, correct?